lib/rules/comma-dangle.js

```diff
@@ -81,49 +81,49 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-
         fixable: "code",
-
-        schema: [
-            {
-                defs: {
-                    value: {
-                        enum: [
-                            "always",
-                            "always-multiline",
-                            "only-multiline",
-                            "never"
-                        ]
-                    },
-                    valueWithIgnore: {
-                        anyOf: [
-                            {
-                                $ref: "#/defs/value"
```

Comment:
Oh I see, I misunderstood what the change was here. I guess the more general question is: is this schema fix necessary, or would the PR still work without this schema fix? (I don't mind modifying the schema in general, but if it requires changes then that might indicate that switching to Ajv would risk breaking third-party rules.)